core/tests/protractor/learnerDashboard.js

```diff
@@ -129,17 +133,17 @@ describe('Learner dashboard functionality', function() {
     // 'In Progress' section.
     player.submitAnswer('Continue', null);
     browser.ignoreSynchronization = true;
-    browser.get(general.LEARNER_DASHBOARD_URL);
+    learnerDashboardPage.get();
     general.acceptAlert();
     browser.waitForAngular();
     general.waitForSystem();
     browser.ignoreSynchronization = false;
-    element(by.css('.protractor-test-incomplete-collection-section')).click();
+    learnerDashboardPage.navigateToIncompleteCollection();
```

Comment:
navigateToIncompleteCollectionSection (you're navigating to a section, not to a collection)

Reply:
Ya, I thought about this, but thought name might become too long. Will change that.

Reply:
Done